plugins/lyrics-genius/back.js

```diff
@@ -17,17 +21,52 @@ module.exports = async (win) => {
 	});
 };
 
+const toggleRomanized = () => {
+	revRomanized = !revRomanized;
+	console.log("Romanized Mode: " + revRomanized);
+};
+
 const fetchFromGenius = async (metadata) => {
-	const queryString = `${cleanupName(metadata.artist)} ${cleanupName(
-		metadata.title
-	)}`;
-	let response = await fetch(
-		`https://genius.com/api/search/multi?per_page=5&q=${encodeURI(queryString)}`
-	);
+	const songTitle = `${cleanupName(metadata.title)}`;
+	const songArtist = `${cleanupName(metadata.artist)}`;
+
+	/* Tried using regex to test the title and artist for East Asian Characters. It works but I realized 
+	some groups are fully English in both title and singer. In this case, the lyrics are  Might improve this in the future.
+	*/
+	let regexEastAsianChars = new RegExp("[\u{3040}-\u{30ff}\u{3400}-\u{4dbf}\u{4e00}-\u{9fff}\u{f900}-\u{faff}\u{ff66}-\u{ff9f}]");
```

Comment:
The regex probably shouldn't be created every time `fetchFromGenius` is called, but rather at the top level

Reply:
I'll address this right now, thanks